content/en/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Poseidon-Firmament - An alternate scheduler
+content_template: templates/concept
+weight: 80
+---
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.6" state="alpha" >}}
+ 
+Poseidon-Firmament scheduler is an alternate scheduler that can be deployed alongside the default Kubernetes scheduler.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+
+## Introduction	
+
+Poseidon is a service that acts as the integration glue for the  [Firmament scheduler](https://github.com/Huawei-PaaS/firmament) with Kubernetes. Poseidon-Firmament scheduler augments the current Kubernetes scheduling capabilities. It incorporates novel flow network graph based scheduling capabilities alongside the default Kubernetes Scheduler. Firmament scheduler models workloads and clusters as flow networks and runs min-cost flow optimizations over these networks to make scheduling decisions.
+
+It models the scheduling problem as a constraint-based optimization over a flow network graph. This is achieved by reducing scheduling to a min-cost max-flow optimization problem. The Poseidon-Firmament scheduler dynamically refines the workload placements.  
+
+Poseidon-Firmament scheduler runs alongside the default Kubernetes Scheduler as an alternate scheduler, so multiple schedulers run simultaneously. 
+
+## Key Advantages
+
+### Flow graph scheduling based Poseidon-Firmament scheduler provides the following key advantages:  
+- Workloads (pods) are bulk scheduled to enable scheduling at massive scale..  
+- Based on the extensive performance test results, Poseidon-Firmament scales much better than the Kubernetes default scheduler as the number of nodes increase in a cluster. This is due to the fact that Poseidon-Firmament is able to amortize more and more work across workloads.  
```

Comment:
I'm not sure I understand what "amortize more and more work across workloads" means. Do you mean "workers (nodes)" instead of workloads?

Reply:
Hi Andrew, what this means is that instead doing filtering and scoring for each individual workload, our scheduler does filtering and scoring once for workloads which have the similar resource requirements. Hence, it amortizes filtering & scoring work across workloads (pods in the case of Kubernetes).